src/BinaryConstraintOps.h

```diff
@@ -65,6 +65,51 @@ inline std::ostream& operator<<(std::ostream& os, BinaryConstraintOp x) {
     return os << toBinaryConstraintSymbol(x);
 }
 
+inline BinaryConstraintOp getEqConstraint(const std::string& type) {
+    switch (type[0]) {
+        case 'f': return BinaryConstraintOp::FEQ;
+        case 'u': return BinaryConstraintOp::EQ;
+        case 'i': return BinaryConstraintOp::EQ; assert("Unsupported type");
```

Comment:
The assert is not particularly useful here. It should either be the default behaviour (i.e. if the type is not used, exit), or the switch could be simplified to `case 'f':  returns FEQ;`, the others all fall through to the standard EQ.